app/core/reservationbook_test.py

```diff
@@ -103,7 +103,7 @@ def mockWaitingMapperDone():
 	assert len(reservationBook.reservationList) == 1
 
 	"""Making sure the user is the first capstone student at the top of the list"""
-	assert reservationBook.reservationList[0].getUser().getName() == 'emir'
+	assert reservationBook.reservationList[0].getUser().getId() == 'emir'
```

Comment:
We can really do this, because it will break how Aspects work. They depend on `getId()` being available.